examples/gno.land/p/demo/dao/dao.gno

```diff
@@ -0,0 +1,32 @@
+package dao
```

Comment:
add a package-level comment to explain the role of this p/

Reply:
Added:

[2f2632b](https://github.com/gnolang/gno/pull/2581/commits/2f2632b79502f45da031de52e1a5e8fa6888c8e5)